src/server/pps/persist/server/rethink_api_server.go

```diff
@@ -137,16 +137,15 @@ func (a *rethinkAPIServer) Close() error {
 // Timestamp cannot be set
 func (a *rethinkAPIServer) CreateJobInfo(ctx context.Context, request *persist.JobInfo) (response *persist.JobInfo, err error) {
 	defer func(start time.Time) { a.Log(request, response, err, time.Since(start)) }(time.Now())
-	if request.JobID != "" {
-		return nil, fmt.Errorf("request.JobID should be unset")
-	}
 	if request.CreatedAt != nil {
```

Comment:
This makes sense having read the rest of the code. How do you feel about going 1 step farther and erroring if it's not set? I'd prefer to only have 1 way to use the API.